Makefile

```diff
@@ -125,7 +125,16 @@ endif
 ENABLE_JIT ?= 0
 $(call set-feature, JIT)
 ifeq ($(call has, JIT), 1)
-OBJS_EXT += jit.o
+OBJS_EXT += jit.o 
+ifeq ("$(shell llvm-config-17 --cflags)", "-I/usr/lib/llvm-17/include  -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS")
```

Comment:
This check might not be invalid for different Linux distributions and macOS (via homebrew).